pkg/operator/ceph/file/mirror/controller.go

```diff
@@ -93,6 +95,7 @@ func newReconciler(mgr manager.Manager, context *clusterd.Context, opManagerCont
 		context:          context,
 		opConfig:         opConfig,
 		opManagerContext: opManagerContext,
+		recorder:         mgr.GetEventRecorderFor("rook-" + controllerName),
 	}
 }
```

Comment:
Aren't these changes supposed to be for block pools only? Why are there also file mirror changes?

Reply:
pr title was miss leading... the intention of the fix is to report events for every controller.